serializers/api/project.rb

```diff
@@ -1,12 +1,15 @@
 # frozen_string_literal: true
 
-require_relative "../base"
-
 class Serializers::Api::Project < Serializers::Base
   def self.base(p)
     {
-      id: p.ubid,
-      name: p.name
+      id: p.id,
+      ubid: p.ubid,
+      path: p.path,
+      name: p.name,
+      credit: p.credit.to_f,
+      discount: p.discount,
```

Comment:
I'm not sure about whether to return credit/discount if they are not set. On one hand, it might create confusion, because many users will not be aware of credits/discounts and this is not good place to learn about them. On the other hand it feels like a bad API design to omit some fields depending on the situation.

Reply:
As we are considering to add credits for onboarding users, let me keep it as it is.